pr_agent/agent/pr_agent.py

```diff
@@ -25,6 +26,7 @@
     "answer": PRReviewer,
     "review": PRReviewer,
     "review_pr": PRReviewer,
+    "best_practices": PRBestPracticesCheck,
```

Comment:
Since this is a fork, let's clearly distinguish BKPR stuff from the rest. Let's prefix everything with `beekeeper_` or simiar